include/vcpkg/vcpkgpaths.h

```diff
@@ -60,6 +61,18 @@ namespace vcpkg
     struct PackageSpec;
     struct Triplet;
 
+    struct ContentAndHash
+    {
+        ContentAndHash(std::string&& content, Hash::Algorithm algo);
```

Comment:
```suggestion
        ContentAndHash(std::string&& content, Hash::Algorithm algo = Hash::Algorithm::Sha1);
```

... and then remove naming Sha1 in all the call sites?

Reply:
We should remove all uses of Sha1; use Sha256 instead.

Reply:
Good point